Makefile

```diff
@@ -49,12 +49,18 @@ build:
 docker_build: download_jars
 	GOOS=linux GOARCH=amd64 make build
 	docker build -t $(APP_NAME) .
+	docker build -t $(APP_NAME)-us-west-2 .	
```

Comment:
You can test to double check, but I think `$APP_NAME` will get injected for both apps as part of the CI build step so the make file doesn't need changes. If you do actually need to build both explicitly, then you would just want to add multiple tags instead of building twice.